tools/generate-imports.mjs

```diff
@@ -93,11 +140,45 @@ async function generateData() {
   );
 }
 
+async function generateHash() {
+  shell.echo('generating hashes');
+  try {
+    const hashMap = `export const hashMap = new Map<string, string>();`;
+    let hashes = [];
+    // get managers list
+    const managers = (
+      await fs.readdir('lib/modules/manager', { withFileTypes: true })
+    )
+      .filter((file) => file.isDirectory())
+      .map((file) => file.name);
+
+    for (const manager of managers) {
+      const hash = await getManagerHash(manager);
+      hashes.push(hash);
+    }
+
+    //add manager hashes to hashMap {key->manager, value->hash}
+    hashes = (await Promise.all(hashes)).map(
+      (hash, index) => `hashMap.set(\n  '${managers[index]}',\n  '${hash}'\n);`
```

Comment:
Do we really need all those newlines ?

Reply:
The generated looks better(formatted) with those new lines. 🙂
No actual reason since are using `.generated.ts` and the files aren't being prettier checked anymore.

Reply:
ok, but prettier would remove the newlines again if used? i would prefer to generate compatible to prettier and maybe use prettier programmatically after generation in future, so we have consistent code style even for generated code.

Reply:
I would think of adding for using prettier generated code and will create a PR soon.